components/react/src/App.js

```diff
@@ -53,6 +54,7 @@ const App = () => {
   return (
     <div className="App">
       <Header />
+      <Experiment />
```

Comment:
```suggestion
```

Reply:
This is resolved with the most recent pushes to the branch